scripts/qa_crowbarsetup.sh

```diff
@@ -4056,6 +4056,29 @@ function onadmin_test_setuphost
     safely $SCRIPTS_DIR/mkcloud setuphost cleanup prepare setupadmin
 }
 
+function oncontroller_test_keystone_password
+{
+    safely openstack token issue
+}
+
+function update_keystone_password
+{
+    local old_password=$(crowbar keystone proposal show default | rubyjsonparse "puts j['attributes']['keystone']['admin']['password']")
+    local updated_password='barcrow'
+    crowbar batch export keystone | ruby -ryaml -e '
+y = YAML.load(ARGF)
+y["proposals"].first["attributes"]["admin"] ||= {}
+y["proposals"].first["attributes"]["admin"]["updated_password"] = "barcrow"
+puts y.to_yaml' > /root/keystone-test-pw-update.yaml
+    safely crowbar batch build < /root/keystone-test-pw-update.yaml
+    safely oncontroller test_keystone_password
+    crowbar batch export keystone | ruby -ryaml -e "
```

Comment:
I guess, we could just `cat /root/keystone-test-pw-update.yaml |` here, or we even leave the password changed, because that is what actual cloud admins should do and thus must be working.

Reply:
Changed it to cat the file. I'd rather not leave the password changed since I can't know if anyone has any personal tooling that depends on the old password but doesn't use the .openrc file, plus as much as possible I think tests should avoid having side effects.